### PR TITLE
Clean documentation

### PR DESCRIPTION
**Pull request by dlitvakb**

### CODE REVIEW

File: contentful_management/client.py [hunk 1]
```diff
@@ -31,28 +31,28 @@
...
-    """Constructs the API Client.
+    """Constructs the API client.
 
-    :param access_token: API Access Token.
-    :param api_url: (optional) URL of the Contentful Target API,
+    :param access_token: API access token.
+    :param api_url: (optional) URL of the Contentful target API,
         defaults to Management API.
-    :param uploads_api_url: (optional) URL of the Contentful Upload Target API,
+    :param uploads_api_url: (optional) URL of the Contentful upload target API,
```
Comment (dborsatto) [inline contentful_management/client.py]: I would remove "target" here

File: contentful_management/client.py [hunk 2]
```diff
@@ -306,9 +306,9 @@ def webhook_health(self, space_id, webhook_id):
         return WebhooksHealthProxy(self, space_id, webhook_id)
 
     def api_keys(self, space_id):
-        """Provides access to Api Keys management methods
+        """Provides access to api key management methods.
```
Comment (dborsatto) [inline contentful_management/client.py]: _API` should probably be capitalized

File: contentful_management/utils.py [hunk 3]
```diff
@@ -51,7 +54,8 @@ def snake_case(a_string):
 
 
 def camel_case(snake_str):
-    """Returns a camel cased version of a string.
+    """
+    Returns a camel cased version of a string.
```
Comment (dborsatto) [inline contentful_management/utils.py]: Should be `camel-cased`

File: contentful_management/utils.py [hunk 4]
```diff
@@ -28,16 +28,19 @@ def emit(self, record):
...
 
     if sys.version_info[0] >= 3:
         return str
     return unicode  # noqa: F821
 
 
 def snake_case(a_string):
-    """Returns a snake cased version of a string.
+    """
+    Returns a snake cased version of a string.
```
Comment (dborsatto) [inline contentful_management/utils.py]: Should be `snake-cased`

File: contentful_management/space_resource_proxy.py [hunk 5]
```diff
@@ -1,10 +1,8 @@
 """
-contentful.space_api_keys_proxy
-~~~~~~~~~~~~~~~~~~~~~~~
+contentful_management.space_api_keys_proxy
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This module implements the SpaceApiKeysProxy class.
-
-API Reference: https://www.contentful.com/developers/docs/references/content-management-api/#/reference/api-keys
+This module implements the SpaceResourceProxy class.
```
Comment (dborsatto) [inline contentful_management/space_resource_proxy.py]: No link to the docs, here?

File: contentful_management/errors.py [hunk 6]
```diff
@@ -87,7 +87,7 @@ class ServiceUnavailableError(HTTPError):
 
 
 def get_error(response):
-    """Gets Error by HTTP Status Code"""
+    """Gets Error by HTTP status code."""
```
Comment (dborsatto) [inline contentful_management/errors.py]: I see that you usually split the comments in multiple lines, like
``` python
"""
Base HTTP error class
"""
```

even when they're a single line, but you don't do it here 🤔 I think that for consistency you should probably choose one of those two alternatives, and stick with it 🙂

File: contentful_management/entry_resource_proxy.py [hunk 7]
```diff
@@ -27,28 +27,28 @@ def _resource_proxy_class(self):
...
-        Creates a resource with a given ID (optional) and attributes for the current Entry.
+        Creates a resource with a given ID (optional) and attributes for the current entry.
         """
 
         return self.proxy.create(resource_id=resource_id, attributes=attributes)
 
     def delete(self, resource_id):
         """
-        Deletes a Resource by ID.
+        Deletes a resource by ID.
```
Comment (dborsatto) [inline contentful_management/entry_resource_proxy.py]: _entry_

File: contentful_management/editor_interfaces_proxy.py [hunk 8]
```diff
@@ -30,14 +30,21 @@ def _resource_class(self):
...
         """
-        Gets the default Editor Interface.
+        Gets the default editor interface.
+        """
+
+        return self.all()
+
+    def default(self):
+        """
+        Gets the default editor interface.
```
Comment (dborsatto) [inline contentful_management/editor_interfaces_proxy.py]: I'm probably missing some context with editor interfaces, but`def default` and _Gets the default editor interface_ imply a single result, but I guess `return self.all()` returns a collection, am I correct?

File: tests/content_type_editor_interfaces_proxy_test.py [hunk 9]
```diff
@@ -33,3 +33,12 @@ def test_content_type_editor_interfaces_proxy_find(self):
...
         self.assertTrue(editor_interface.id)
+
+    @vcr.use_cassette('fixtures/editor_interface/all.yaml')
+    def test_content_type_editor_interfaces_proxy_default(self):
+        proxy = ContentTypeEditorInterfacesProxy(CLIENT, PLAYGROUND_SPACE, 'foo')
+
+        editor_interface = proxy.default()
+
+        self.assertTrue(editor_interface)
+        self.assertTrue(editor_interface.id)
```
Comment (dborsatto) [inline tests/content_type_editor_interfaces_proxy_test.py]: This test is related to the single vs collection question I have 🙂

**Comment by dborsatto** [comment]
Overall, is there a rationale is writing comments in a single line
```
"""This is a comment"""
```
versus splitting them in multiple lines?
```
"""
This is a comment
"""
```

File: contentful_management/space_resource_proxy.py (hunk 5 above)
Comment (dlitvakb) [inline contentful_management/space_resource_proxy.py]: This is a base class, it doesn't have specific documentation.

File: contentful_management/entry_resource_proxy.py (hunk 7 above)
Comment (dlitvakb) [inline contentful_management/entry_resource_proxy.py]: This is the base class for resources that are dependant on an entry, so `resource` is generic in this case.

File: contentful_management/editor_interfaces_proxy.py (hunk 8 above)
Comment (dlitvakb) [inline contentful_management/editor_interfaces_proxy.py]: The 3 methods return a single `default` editor interface, it still has the 3 methods due to editor interfaces previously not being a singleton endpoint as it is now.

File: contentful_management/entry_resource_proxy.py (hunk 7 above)
Comment (dborsatto) [inline contentful_management/entry_resource_proxy.py]: Got it 👍

**Pull request merged**